users/common/utils.py

```diff
@@ -18,23 +21,30 @@ def generate_random_phone_number(length):
 
 
 def raise_serializer_exception(error_message):
+    logger.warning(f"Serializer exception with message: {error_message}")
     raise serializers.ValidationError(error_message)
 
 
 def raise_custom_validation_exception(error_message, error_code):
+    logger.warning(f"Custom valid exception with message: {error_message} and error code: {error_code}")
     raise CustomValidationError(error_message, error_code)
 
 
 def prepare_proper_exception(call_instance, error_message, error_code):
+    logger.info("Preparing exception")
     if "serializers" in str(type(call_instance)):
+        logger.debug(f"Serializer in call instance: {call_instance}")
```

Comment:
This one is OK but message should be something like `Preparing exception caught in serializer instance {call_instance}`.

Reply:
Done